misc.c

```diff
@@ -2092,6 +2092,29 @@ GC_write(int fd, const char *buf, size_t len)
 #  define DECL_BUF_AND_PRINTF_TO(buf, format) const char *buf = (format)
 #endif /* GC_DISABLE_SNPRINTF */
 
+void 
+GC_CALL GC_set_incremental(GC_bool incremental)
```

Comment:
Define `GC_disable_incremental(void)` instead. Because we already have `GC_enable_incremental()`.